include/versions.h

```diff
@@ -0,0 +1,24 @@
+#ifndef VERSIONS_H
+#define VERSIONS_H
+
+// OOT versions in build order
+#define OOT_NTSC_1_2 1
+#define OOT_GC_JP 2
+#define OOT_GC_JP_MQ 3
+#define OOT_GC_US 4
+#define OOT_GC_US_MQ 5
+#define OOT_GC_EU_MQ_DBG 6
+#define OOT_GC_EU 7
+#define OOT_GC_EU_MQ 8
+#define OOT_GC_JP_CE 9
+
+#define FAULT_YS 1 // in Yoshi's Story
```

Comment:
It would be nice to be able to build N64 versions with the better fault handler, so have a layer of indirection between "n64" and "use either fault handler". Here it's the `FAULT_VERSION`

I'd agree we don't need to acknowledge yoshi's story. At one point I refered to the oot-n64 and oot-gc fault versions as v1 and v2, but that'd make yoshi's story's "v0". So in case someone would suggest calling the oot handlers v1 and v2, I'd rather not use numbers because this

I'll remove it now though

Reply:
I see, I guess that also explains the dummy implementations of e.g. Fault_WaitForInput that you added initially. I'm not convinced that this should be a special case compared to other N64/GC version differences, I feel like we should use PLATFORM_N64/PLATFORM_GC here like the rest of the code and it's the job of modding repos like HackerOOT can make it easier to swap implementations. Or, maybe wait until later until it's clearer which features deserve their own flags (e.g. map select) and which should just be version-specific changes

(Also I don't think this layer of indirection actually works right now because of the `ifeq ($(PLATFORM),N64)` in the Makefile)

Reply:
After more thought I'm fine with this. I think `FAULT_OOTN64` -> `FAULT_N64` though and renumber from 1 though to fully excise Yoshi's Story